### PR TITLE
refactor(page): remove toolbar from page component header

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/1231

### CODE REVIEW

File: src/patternfly/components/Page/page-header-tools-item.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-page__header-tools-item{{#if page-header-tools-item--modifier}} {{page-header-tools-item--modifier}}{{/if}}"
```
Comment (matthewcarleton) [inline src/patternfly/components/Page/page-header-tools-item.hbs]: I don't see this being used, do we need to keep it?

File: src/patternfly/demos/Page/page-template-header-tools-elements.hbs [hunk 2]
```diff
@@ -1,29 +1,21 @@
...
-      {{/toolbar-item}}
-    {{/toolbar-group}}
-    {{#> toolbar-group}}
-      {{#> toolbar-item toolbar-item--modifier="pf-m-mobile"}}
-        {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Overflow actions"'}}
-          {{> button-icon button-icon--type="ellipsis-v"}}
-        {{/button}}
-      {{/toolbar-item}}
-      {{#> toolbar-item toolbar-item--modifier="pf-m-user pf-screen-reader"}}
+    <div class="pf-m-icons pf-screen-reader">
```
Comment (matthewcarleton) [inline src/patternfly/demos/Page/page-template-header-tools-elements.hbs]: did you forget to add the `.pf-c-page__header-tools-group` here? I think you'll need it to get the proper spacer.

File: src/patternfly/demos/Page/page-template-header-tools-elements.hbs [hunk 3]
```diff
@@ -1,29 +1,21 @@
...
+      {{#> button button--modifier="pf-m-plain" pagelayout-icon--bell="true" button--attribute='aria-label="Notifications"'}}
```
Comment (matthewcarleton) [inline src/patternfly/demos/Page/page-template-header-tools-elements.hbs]: do you need `pagelayout-icon--bell="true"`

Comment (mcoker) [reply src/patternfly/demos/Page/page-template-header-tools-elements.hbs]: nope!

File: src/patternfly/demos/Page/page-template-header-tools-elements.hbs (hunk 2 above)
Comment (mcoker) [reply src/patternfly/demos/Page/page-template-header-tools-elements.hbs]: yep!!

File: src/patternfly/components/Page/page-header-tools-item.hbs (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/Page/page-header-tools-item.hbs]: nope!

**Pull request merged**